ciphey/basemods/Decoders/galactic.py

```diff
@@ -0,0 +1,98 @@
+# community
+# by https://github.com/lukasgabriel
+
+from typing import Optional, Dict, List
+
+from ciphey.iface import Config, ParamSpec, T, U, Decoder, registry
+
+
+@registry.register
+class Galactic(Decoder[str, str]):
+    def decode(self, ctext: T) -> Optional[U]:
+        """
+        Takes a string written in the 'Standard Galactic Alphabet' 
+        (aka Minecraft Enchanting Table Symbols) and translates it to ASCII text.
+        """
+
+        result = ""
+        galactic_letters = [
+            "ᔑ",
+            "ʖ",
+            "ᓵ",
+            "↸",
+            "ᒷ",
+            "⎓",
+            "⊣",
+            "⍑",
+            "╎",
+            "⋮",
+            "ꖌ",
+            "ꖎ",
+            "ᒲ",
+            "リ",
+            "𝙹",
+            "!",
+            "ᑑ",
+            "∷",
+            "ᓭ",
+            "ℸ",
+            "⚍",
+            "⍊",
+            "∴",
+            "|",
+            "⨅",
+        ]
+        letters = list("abcdefghijklmnopqrstuvwyz")
```

Comment:
We have this in CipheyDists too, also, make sure your input text is lowercase, or the output text is uppercase.

Is there uppercase galactic language? As a convention, Ciphey returns lower case for stuff we're not sure on I believe :P

Reply:
Galactic language is lowercase only